File: Semestrul 4/SDI/lab08/TriathlonNetworking/DTOs/ResultDTO.cs
using Triathlon.Model;
using System;

namespace Triathlon.Network.Dto
{
    [Serializable]
    public class ResultDTO
    {
        private long id;
        private long eventId;
        private long participantId;
        private Discipline discipline;
        private int points;

        public ResultDTO() { }

        public ResultDTO(long id, long eventId, long participantId, Discipline discipline, int points)
        {
            this.id = id;
            this.eventId = eventId;
            this.participantId = participantId;
            this.discipline = discipline;
            this.points = points;
        }

        public long Id { get { return id; } set { id = value; } }
        public long EventId { get { return eventId; } set { eventId = value; } }
        public long ParticipantId { get { return participantId; } set { participantId = value; } }
        public Discipline Discipline { get { return discipline; } set { discipline = value; } }
        public int Points { get { return points; } set { points = value; } }
    }
}

File: Semestrul 4/SDI/lab14/java/TriathlonNetworking/src/main/java/triathlon/network/dto/ResultDTO.java
package triathlon.network.dto;

import triathlon.model.Discipline;
import java.io.Serializable;

public class ResultDTO implements Serializable {
    private Long id;
    private Long eventId;
    private Long participantId;
    private Discipline discipline;
    private int points;

    public ResultDTO() {}

    public ResultDTO(Long id, Long eventId, Long participantId, Discipline discipline, int points) {
        this.id = id;
        this.eventId = eventId;
        this.participantId = participantId;
        this.discipline = discipline;
        this.points = points;
    }

    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }
    public Long getEventId() { return eventId; }
    public void setEventId(Long eventId) { this.eventId = eventId; }
    public Long getParticipantId() { return participantId; }
    public void setParticipantId(Long participantId) { this.participantId = participantId; }
    public Discipline getDiscipline() { return discipline; }
    public void setDiscipline(Discipline discipline) { this.discipline = discipline; }
    public int getPoints() { return points; }
    public void setPoints(int points) { this.points = points; }
}